### DISCUSSION TITLE
LYWSD03MMC with a even bigger battery

### DISCUSSION THREAD

**Discussion by DavidStacer** [pvvx/ATC_MiThermometer#434] [Show and tell]
This is an even bigger battery using a CR123a 3v lithium battery.  Why?  I had a pack of 6 of the CR123a left and I have no other use for them. CR123a have 1500mAh capacity and have a wide operating temperature range -40C to 60C

I have a slightly different style with a CR123a deployed outside for a couple of years.  That was a hard modification to make.  This modification is much easier.  [refer to my previous post](https://github.com/pvvx/ATC_MiThermometer/discussions/396) to see how to cut into the back.   

Take off the black cover foam rubber and they give you a nice center mark.  Drill out the middle. Then use a rotary tool to make the hole bigger to fit the 3D printed part.

I designed a 3D printed holder with a grove for a wire. Using a 8mm x 9mm spring battery terminal in the end.  Please note this is the positive (button top) of the battery that goes on the spring. The flat end (negative) of the battery will go inside the temperature sensor and touch the metal contacts on the black PCB. 

This photo shows a earlier blue prototype sticking through the back. In the final version the 3D print extends down to the end of the battery.

![BackPlate](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/504e1b74-038b-4e80-9770-1f9c89fde076)

This photo shows how the cut back cover should look like
![CutCase](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/fcc5e1db-1d6f-4725-a0a5-a8568d80258d)

Battery holder with wire going to the positive end of the battery
![Holder1](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/fb5d8a38-8a2f-458f-a937-cba77fa70ff2)

Battery holder view from the side. The small metal tab will touch the one tab inside the sensor where the original coin cell positive outside case connected.
![Holder2](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/e9424ec7-af66-43ef-a420-caf23043fa32)

Detail look at the spring and wire.
![WireTerminal](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/035dbbf4-c548-4fe0-abfe-1b873e0b751a)

Final result photos - the new battery holder is glued in place to the back cover
![Final2](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/559e2128-1865-476b-a0bd-60e7a4e12d6d)
![Final1](https://github.com/pvvx/ATC_MiThermometer/assets/16613848/5915080e-efa3-4e01-8b8f-4804ad0f42f0)

I designed the 3D printed battery holder using Tinkercad.  Link to the shared project  (https://www.tinkercad.com/things/1WUskKzDATe-lywsd03mmc-battery-extension)

**Comment by cervalx** [comment]
That's a big battery! Thanks for the work on the 3D files.

**Comment by mr-sneezy** [comment]
Nice work. You know it will probably outlast the Bluetooth protocols life itself :)

**Comment by pvvx** [comment]
I placed smaller batteries with a thermometer in inaccessible places during the construction of the building. For several years they reported what the humidity and temperature were there... 
This showed that everything was done well, there was no condensation and the thermal insulation was good. 
When the building is demolished, these thermometers will be found. :)
